payload/Library/Scripts/Install or Defer.sh

```diff
@@ -12,8 +12,8 @@
 #                   the system restarts automatically.
 #         Authors:  Mario Panighetti and Elliot Jordan
 #         Created:  2017-03-09
-#   Last Modified:  2020-07-06
-#         Version:  3.0.2
+#   Last Modified:  2020-11-17
+#         Version:  3.0.3
```

Comment:
I think a change as major as changing plist key names should be accompanied by a version bump from 3.0.2 to 4.0.0 (indicating [non-backwards-compatible](https://semver.org/)).

Reply:
I was saving the major version change for eventually dropping High Sierra support, but that makes perfect sense. :) Bumped to 4.0.